tensorboard/components/vz_bar_chart/vz-bar-chart.ts

```diff
@@ -248,39 +242,42 @@ class BarChart {
     // Filter down the data so each value contains 0 or 1 elements in the array,
     // which correspond to the value of the closest clustered bar (e.g. 'c').
     // This generates {series0: Array(1), series1: Array(0)}.
-    let points = _.mapValues(
+    let bars = _.mapValues(
         this.data,
         allValuesForSeries =>
             _.filter(allValuesForSeries, elt => elt.x == hoveredClass));
 
     // Remove the keys that map to an empty array, and unpack the array.
     // This generates {series0: { x: 'c', y: 3 }}
-    points = _.pick(points, val => val.length > 0);
-    const singlePoints = _.mapValues(points, val => val[0]);
+    bars = _.pick(bars, val => val.length > 0);
+    const singleBars = _.mapValues(bars, val => val[0]);
 
     // Rearrange the object for convenience.
     // This yields: [{key: 'series0', value: { x: 'c', y: 3 }}, ]
-    const formattedPoints = d3.entries(singlePoints);
+    const barEntries = d3.entries(singleBars);
 
-    // Bind the points data structure to the tooltip.
+    // Bind the bars data structure to the tooltip.
     const rows = this.tooltip.select('tbody')
                    .html('')
                    .selectAll('tr')
-                   .data(formattedPoin ts)
+                   .data(barEntries)
                    .enter()
                    .append('tr');
 
     rows.style('white-space', 'nowrap');
+    rows.classed('closest', d => d.key == hoveredSeries)
     const colorScale = this.colorScale;
     rows.append('td')
         .append('div')
         .classed('swatch', true)
         .style('background-color', d => colorScale.scale(d.key));
     _.each(tooltipColumns, (column) => {
       rows.append('td').text((d) => {
-        // TypeScript cannot detect the type of data bound to the elements.
-        const point = d as any as vz_chart_helpers.Point;
-        return column.evaluate(point);
+        // Convince TypeScript to let us pass off a key-value entry of value
+        // type Bar as a Point since that's what TooltipColumn.evaluate wants.
+	// TODO(nickfelt): reconcile the incompatible typing here
```

Comment:
This comment makes more sense! nit: Extra space needed before this line.

Reply:
Done.